build.php

```diff
@@ -0,0 +1,125 @@
+<?php
```

Comment:
You can delete this file as it only needs to be used once when the boilerplate is first cloned.

Reply:
Done! Ref: https://github.com/impress-org/givewp-facebook-pixel-tracking/pull/8/commits/f870fdadef3abe81ac2ed01da43f2d69b2b60c74